pkgs/development/python-modules/cle/default.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, buildPythonPackage
+, cffi
+, fetchFromGitHub
+, minidump
+, nose
+, pefile
+, pyelftools
+, pytestCheckHook
+, pythonOlder
+, pyvex
+, pyxbe
+, sortedcontainers
+}:
+
+buildPythonPackage rec {
+  pname = "cle";
+  version = "9.0.5739";
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "angr";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0vdf3727z09h5n2m7j9bakh3x2531k6270gbw6w67apvdygq298k";
+  };
+
+  # Binary files from https://github.com/angr/binaries (only used for testing and only here)
+  binaries = fetchFromGitHub {
+    owner = "angr";
+    repo = "binaries";
+    rev = "v${version}";
+    sha256 = "1p1rsv11jc0byzw532m12m5sn2akrhbibm2379cg3xgq4yi6hx9l";
+  };
+
+  propagatedBuildInputs = [
+    cffi
+    minidump
+    pefile
+    pyelftools
+    pyvex
+    pyxbe
+    sortedcontainers
+  ];
+
+  checkInputs = [
+    nose
+    pytestCheckHook
+  ];
+
+  # Place test binaries in the right location
+  preCheck = ''
+    cp -r ${binaries} /build/binaries
```

Comment:
I am not sure of we want to use absolute paths here.

Reply:
The tests are expecting that the files are there. We could move them and patch the location here and later in `angr`.

Reply:
I think this directory is configurable so it shouldn't be hardcoded.

Reply:
The tests which are working with the content provided by `binaries` contains

```python
TEST_BASE = os.path.join(os.path.dirname(os.path.realpath(__file__)),
    os.path.join('..', '..', 'binaries'))
```

Reply:
I think using a relative path or patching the location is preferable.

Reply:
@fabaff 
Here is what work for me: https://github.com/angr/nixpkgs/blob/master/pkgs/python-modules/pyxbe/default.nix .